packages/odyssey-design-tokens/src/color/border.json

```diff
@@ -0,0 +1,10 @@
+{
+  "color": {
+    "border": {
+      "ui": { "value": "{color.palette.neutral.500.value}" },
+      "display": { "value": "{color.palette.neutral.200.value}" },
+      "danger": { "value": "{color.danger.base.value}" },
+      "primary": { "value": "{color.primary.base.value}" }
```

Comment:
building from the above comment this is a good example of a how I'd like to see functional colors mapped through to other tokens that share the naming scheme